libs/components/forms/testing/src/form-error/form-errors-harness.ts

```diff
@@ -35,84 +24,73 @@ export class SkyFormErrorsHarness extends SkyComponentHarness {
   }
 
   /*
-   * Gets total number of errors.
-   */
-  public async getNumberOfErrors(): Promise<number> {
-    return (await this.#getFormError()).length;
-  }
-
-  /*
-   * Gets if the required error has triggered.
+   * Whether the required field is set to an empty value.
    */
   public async hasRequiredError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-required',
-    );
+    return this.#hasErrorOfType('required');
   }
 
   /*
-   * Gets if the maximum length error has triggered.
+   * Whether the field has more characters than is allowed.
    */
   public async hasMaxLengthError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-maxlength',
-    );
+    return this.#hasErrorOfType('maxlength');
   }
 
   /*
-   * Gets if the minimum length has triggered.
+   * Whether the field has fewer characters than is allowed.
    */
   public async hasMinLengthError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-minlength',
-    );
+    return this.#hasErrorOfType('minlength');
   }
 
   /*
-   * Gets if the character count error has triggered.
+   * Whether the field has exceeded its maximum character count allowance.
    */
   public async hasCharacterCountError(): Promise<boolean> {
-    return (await this.#getFormErrorsClasses()).includes(
-      'sky-form-error-character-counter',
-    );
+    return this.#hasErrorOfType('character-counter');
   }
 
   /*
-   * Gets if the date error has triggered.
+   * Gets the field is set to an invalid date.
```

Comment:
```suggestion
   * Whether the field is set to an invalid date.
```
Shouldn't we change "Get" here to "Whether" like in the other JSDocs updates?